lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -132,20 +170,27 @@ async function begin() {
   const invertMatch = argv.invertMatch;
   const testDefns = getDefinitionsToRun(allTestDefns, requestedTestIds, {invertMatch});
 
-  const takeNetworkRequestUrls = () => {
-    return server.takeRequestUrls();
-  };
-
-  const options = {jobs, retries, isDebug: argv.debug, lighthouseRunner, takeNetworkRequestUrls};
-
   let isPassing;
+  let server;
+  let serverForOffline;
+  let takeNetworkRequestUrls = undefined;
+
   try {
-    server.listen(10200, 'localhost');
-    serverForOffline.listen(10503, 'localhost');
-    isPassing = (await runSmokehouse(testDefns, options)).success;
+    // If running the core tests, spin up the test server.
+    if (testDefnPath === coreTestDefnsPath) {
```

Comment:
maybe we should expose the static server but make the folder that is served configurable? altho we'd be making the static server query params and such part of public interface, maybe not worth it

Reply:
> maybe we should expose the static server but make the folder that is served configurable? altho we'd be making the static server query params and such part of public interface, maybe not worth it

yeah, looking at the publisher-ads script just running their own server convinced me it wasn't worth it. I think most people would never need all the many options we have in there, so using a third-party one seems like an easier path for most.